BaseLib/ConfigTreeNew.h

```diff
@@ -194,8 +194,12 @@ class ConfigTreeNew final
 
     ConfigTreeNew() = delete;
```

Comment:
Removed.
Actually, because of the move-assignment, now default-construction would in principle be possible. But I refrain from it, because then it would be possible to construct instances that are in an invalid state (i.e. no reading from them is possible) until they are move-assigned to.